packages/jest-worker/src/Farm.ts

```diff
@@ -23,27 +23,31 @@ import {
 
 export default class Farm {
   private _computeWorkerKey: FarmOptions['computeWorkerKey'];
+  private _workerSchedulingPolicy: FarmOptions['workerSchedulingPolicy'];
   private _cacheKeys: Record<string, WorkerInterface>;
   private _callback: Function;
   private _locks: Array<boolean>;
   private _numOfWorkers: number;
-  private _offset: number;
+  private _offset: number = 0;
   private _taskQueue: TaskQueue;
 
   constructor(
     numOfWorkers: number,
     callback: Function,
```

Comment:
```suggestion
    private numOfWorkers: number,
    private callback: Function,
```

and remove from line 28 & 30